src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -904,6 +904,88 @@ describe("Artwork type", () => {
     })
   })
 
+  describe("#canRequestLotConditionsReport", () => {
+    const query = `
+      {
+        artwork(id: "richard-prince-untitled-portrait") {
+          canRequestLotConditionsReport
+        }
+      }
+    `
+
+    it("is true if the artwork's sale is an auction, live, and has lot conditions report enabled", () => {
+      context.saleLoader = sinon.stub().returns(
+        Promise.resolve({
+          live_start_at: moment.utc("2020-12-31").format(),
```

Comment:
Are we sure about this? I was under the impression that we wanted to enable the ability for a user to request lot condition reports _even before_ a sale goes live. Can you double check this?

Reply:
Actually, this would mean that we _never_ show the button for LAI sales, which I'm pretty certain isn't right...

The reaction code is using [`is_biddable`](https://github.com/artsy/metaphysics/blob/b5ba5830706151e65f587a3c0597a5d0d16dfff3/src/schema/v1/sale_artwork.ts#L33) to determine whether to display the button or not...

Reply:
The `is_biddable` function you referred to is a function that is used for `SaleArtwork` type. The actual flag we use in Reaction is [the one in the `Artwork` type](https://github.com/artsy/metaphysics/blob/476198a1f1db35261abf55272aa6c4d27f7f44c9/src/schema/v2/artwork/index.ts#L327-L333), which is different. Obviously it is a question, but I think what we should be doing as part of the story is to respect the existing behaviour and add the extra check with `lot_conditions_report_enabled`, which is what's in this PR.

Reply:
Also we only check the presence of the `live_start_at`, which means the flag would be `true` even for sales that haven't started yet.

Reply:
We have discussed on zoom and here are our conclusions:

 * We want to keep the ability to show this button for all sales (live and OO)
 * `Sale#live` in gravity is a very confusingly named attribute as it really means “open for bidding” and has nothing to do with live auction integration
 * We want to extend but not change the existing rules in Reaction and Emission: show the button if the sale is an auction, is “live” (aka open for bidding) and that sale has the Ohm checkbox enabled.